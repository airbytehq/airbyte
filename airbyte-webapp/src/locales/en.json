{
  "webapp.cannotReachServer": "Cannot reach server. The server may still be starting up.",

  "sidebar.sources": "Sources",
  "sidebar.destination": "Destination",
  "sidebar.help": "Help",
  "sidebar.docs": "Docs",

  "form.continue": "Continue",
  "form.emailOptional": "Your email (optional)",
  "form.email.placeholder": "you@company.com",
  "form.email.error": "This email address doesn’t seem correct.",
  "form.empty.error": "Empty field",
  "form.selectConnector": "Select a connector",
  "form.searchName": "search by name...",
  "form.noResult": "No result",
  "form.sourceName.placeholder": "Your source name",
  "form.destinationName.placeholder": "Your destination name",
  "form.connectionName.placeholder": "Name",
  "form.name": "Name",
  "form.sourceName.message": "Pick a name to help you identify this source in Airbyte",
  "form.destinationName.message": "Pick a name to help you identify this destination in Airbyte",
  "form.connectionName.message": "Pick a name to help you identify this source in Airbyte",
  "form.sourceType": "Source type",
  "form.destinationType": "Destination type",
  "form.connectionType": "Connector type",
  "form.frequency": "Sync frequency",
  "form.frequency.placeholder": "Select a frequency",
  "form.frequency.message": "Set how often Airbyte attempts to replicate data.",
  "form.dataSync": "Select the data you want to sync",
  "form.dataSync.message": "You’ll be able to change this later on.",
  "form.cancel": "Cancel",
  "form.delete": "Delete",
  "form.saveChanges": "Save changes",
  "form.discardChanges": "Discard changes",
  "form.every": "Every {value}",
  "form.testingConnection": "Testing connection...",
  "form.successTests": "All connection tests passed!",
  "form.failedTests": "The connection tests failed.",
  "form.host": "Host",
  "form.port": "Port",
  "form.user": "User",
  "form.password": "Password",
  "form.dbname": "DB Name",
  "form.postgres_host": "Host",
  "form.postgres_port": "Port",
  "form.postgres_username": "User",
  "form.postgres_password": "Password",
  "form.postgres_database": "DB Name",
  "form.postgres_schema": "Schema",
  "form.filter_dbs": "Filter DBs",
  "form.serviceAccount": "Service Account",
  "form.project_id": "Project ID",
  "form.dataset_id": "Dataset ID",
  "form.validate_records": "Validate Records",
  "form.credentials_json": "Credentials JSON",
  "form.disable_collection": "Disable Collection",
  "form.replication_method": "Replication Method",
  "form.validationError": "The received form did not pass validation",
  "form.someError": "Sorry. Something went wrong... ",
  "form.changesSaved": "Your changes were saved!",

  "preferences.title": "Specify your preferences",
  "preferences.anonymizeUsage": "Anonymize usage data collection",
  "preferences.collectData": "We collect data only for product improvements, see the <docs>docs</docs>.",
  "preferences.news": "News and feature updates",
  "preferences.security": "Security updates",
  "preferences.anonymizeData": "Anonymize my usage data.",
  "preferences.featureUpdates": "Receive feature updates.",
  "preferences.unsubcribeAnyTime": " You can unsubcribe any time.",
  "preferences.securityUpdates": "Receive emails about security updates.",

  "onboarding.title": "Let’s set up your first connection.",
  "onboarding.subtitle": "Connections are automated data pipelines that sync data from a source to a destination.",
  "onboarding.createSource": "Create a source",
  "onboarding.sourceSetUp": "Set up the source",
  "onboarding.sourceSetUp.buttonText": "Set up source",
  "onboarding.createDestination": "Create a destination",
  "onboarding.destinationSetUp": "Set up the destination",
  "onboarding.destinationSetUp.buttonText": "Set up destination",
  "onboarding.setUpConnection": "Set up connection",
  "onboarding.setConnection": "Set up the connection",
  "onboarding.instructionsLink": "See instructions how to connect {name}",
  "onboarding.fetchingSchema": "We are fetching the schema of your data source.\nThis should take less than a minute...",

  "sources.newSource": "+ New source",
  "sources.newSourceTitle": "New Source",
  "sources.selectSource": "Select a source",
  "sources.name": "Name",
  "sources.progress": "In progress",
  "sources.connector": "Connector",
  "sources.frequency": "Frequency",
  "sources.lastSync": "Last Sync",
  "sources.enabled": "Enabled",
  "sources.launch": "Launch",
  "sources.status": "Status",
  "sources.schema": "Schema",
  "sources.settings": "Settings",
  "sources.syncHistory": "Sync history",
  "sources.syncNow": "Sync now",
  "sources.source": "Source",
  "sources.disabled": "Disabled",
  "sources.noSync": "No sync yet",
  "sources.emptySchema": "Schema is empty",
  "sources.noSources": "Source list is empty",
  "sources.sourceSettings": "Source Settings",
  "sources.dataDelete": "No data will be deleted from your source.",
  "sources.deleteSource": "Delete this source",
  "sources.deleteConfirm": "Confirm source deletion",
  "sources.failed": "Failed",
  "sources.completed": "Completed",
  "sources.pending": "Pending",
  "sources.running": "Running",
  "sources.cancelled": "Cancelled",
  "sources.emptyLogs": "Empty data",
  "sources.hour": "{hour}h ",
  "sources.minute": "{minute}m ",
  "sources.second": "{second}s",
  "sources.deleteModalText": "This can not be un-done without a full re-sync. Note that:\n  - All past logs and configurations will be deleted\n  - Updates of new data will stop\n  - No existing data in the destination will be altered",

  "destination.destinationSettings": "Destination Settings"
}
