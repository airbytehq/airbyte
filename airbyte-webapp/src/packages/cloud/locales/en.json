{
  "login.haveAccount": "Already have an account?",
  "login.DontHaveAccount": "Donâ€™t have an account?",
  "login.login": "Log in",
  "login.signup": "Sign up",
  "login.signup.submitButton": "Sign up",
  "login.loginTitle": "Log in to Airbyte",
  "login.resendEmail": "Didnâ€™t receive the email? Send it again",
  "login.yourEmail": "Your work email*",
  "login.inviteEmail": "For security, re-enter your invite email*",
  "login.yourEmail.placeholder": "work.email@example.com",
  "login.yourEmail.notFound": "User not found",
  "login.unknownError": "An unknown error has occurred",
  "login.password": "Enter your password*",
  "login.password.placeholder": "Enter a strong password",
  "login.yourPassword": "Enter your password*",
  "login.yourPassword.placeholder": "Your password",
  "login.forgotPassword": "Forgot your password",
  "login.backLogin": "Back to Log in",
  "login.createPassword": "Create a password*",
  "login.resetPassword": "Reset your password",
  "login.resetPassword.emailSent": "A password reset email has been sent to you",
  "login.activateAccess": "Activate your <hl>14-day free trial</hl>",
  "login.activateAccess.subtitle": "ðŸŽ‰ Special <launch>launch offer</launch> of <sum>$1,000</sum> worth of free usage!",
  "login.activateAccess.submitButton": "Sign up",
  "login.fullName": "Full name*",
  "login.fullName.placeholder": "Christopher Smith",
  "login.companyName": "Company name*",
  "login.companyName.placeholder": "Acme Inc.",
  "login.subscribe": "Receive community and feature updates. You can unsubscribe any time. ",
  "login.security": "By using the service, you agree to to our <terms>Terms of Service</terms> and <privacy>Privacy\u00a0Policy</privacy>.",
  "login.inviteTitle": "Invite access",
  "login.inviteLinkExpired": "This invite link expired. A new invite link was sent to your email.",
  "login.inviteLinkInvalid": "This invite link is no longer valid.",
  "login.rightSideFrameTitle": "More about Airbyte",
  "login.quoteText": "Airbyte has cut <b>months of employee hours off</b> of our ELT pipeline development and delivered usable data to us in hours instead of weeks. We are excited for the future of Airbyte, enthusiastic about their approach, and optimistic about our future together.",
  "login.quoteAuthor": "Micah Mangione",
  "login.quoteAuthorJobTitle": "Director of Technology",

  "confirmResetPassword.newPassword": "Enter a new password",
  "confirmResetPassword.success": "Your password has been reset. Please log in with the new password.",
  "confirmResetPassword.error.expiredActionCode": "The password reset link has expired. Please request password reset again.",
  "confirmResetPassword.error.invalidActionCode": "The password reset link is invalid. Please double check the reset email.",
  "confirmResetPassword.error.weakPassword": "Please enter a more sophisticated new password.",
  "confirmResetPassword.error.default": "Password reset failed: ",

  "login.selfhosting": "Interested in self-hosting?",
  "login.opensource": "Open-source",
  "login.deployInfrastructure": "Deploy the open-source version on your own infrastructure",

  "workspaces.title": "Select a workspace",
  "workspaces.subtitle": "Workspaces let you collaborate with team members and share sources across your team under a shared billing account.",
  "workspaces.create": "Create workspace",
  "workspaces.createNew": "Create new workspace",
  "workspaces.loading": "Loadingâ€¦",

  "settings.accountSettings.logoutLabel": "Sign out",
  "settings.accountSettings.logoutText": "Sign out",
  "settings.accountSettings.firstName": "First name",
  "settings.accountSettings.enterPassword": "Enter current password",
  "settings.accountSettings.firstName.placeholder": "Christopher",
  "settings.accountSettings.fullName": "Full name",
  "settings.accountSettings.fullName.placeholder": "Christopher Smith",
  "settings.accountSettings.lastName": "Last name",
  "settings.accountSettings.lastName.placeholder": "Smith",
  "settings.accountSettings.email": "Email",
  "settings.accountSettings.updateEmail": "Update Email",
  "settings.accountSettings.password": "Password",
  "settings.accountSettings.currentPassword": "Current password",
  "settings.accountSettings.currentPassword.placeholder": "Current password",
  "settings.accountSettings.newPassword": "New password",
  "settings.accountSettings.newPasswordConfirmation": "Password confirmation",
  "settings.accountSettings.error.newPasswordMismatch": "New password and confirmation do not match",
  "settings.accountSettings.error.newPasswordSameAsCurrent": "New password is the same as the current one",
  "settings.accountSettings.updatePasswordError": "Password update failed: ",
  "settings.accountSettings.updatePasswordSuccess": "Your password has been updated!",
  "settings.accountSettings.updatePassword": "Update password",
  "settings.userSettings": "User settings",
  "settings.workspaceSettings": "Workspace settings",
  "settings.generalSettings": "General Settings",
  "settings.generalSettings.changeWorkspace": "Change Workspace",
  "settings.generalSettings.form.name.label": "Workspace name",
  "settings.generalSettings.form.name.placeholder": "Workspace name",
  "settings.generalSettings.deleteLabel": "Delete your workspace",
  "settings.generalSettings.deleteText": "Delete",
  "settings.accessManagementSettings": "Access Management",
  "userSettings.table.title": "Current users",
  "userSettings.table.column.fullname": "Full Name",
  "userSettings.table.column.email": "Email",
  "userSettings.table.column.role": "Role",
  "userSettings.table.column.action": "Action",
  "userSettings.button.addNewUser": "New user",
  "userSettings.user.remove": "Remove",
  "modals.addUser.title": "Add new users",
  "modals.addUser.email.label": "Email address",
  "modals.addUser.role.label": "Role",
  "modals.addUser.role.placeholder": "Select a role",
  "modals.addUser.button.addUser": "Add another user",
  "modals.addUser.button.cancel": "Cancel",
  "modals.addUser.button.submit": "Send invitation",
  "workspaces.viewAllWorkspaces": "View all workspaces",
  "settings.accessManagement.roleViewers": "<b>Viewers</b> are in read-only and cannot edit or add connections.",
  "settings.accessManagement.roleEditors": "<b>Editors</b> can edit connections",
  "settings.accessManagement.roleAdmin": "<b>Admin</b> can also manage users",

  "credits.date": "Date",
  "credits.amount": "Credits",

  "credits.usagePerConnection": "Usage per connection",
  "credits.connection": "Connection",
  "credits.usage": "Usage",
  "credits.noData": "Data is empty",
  "credits.creditsProblem.negative_within_grace_period": "Your workspace is out of credits! Please add more credits to make sure your syncs keep running!",
  "credits.creditsProblem.negative_beyond_grace_period": "Youâ€™re out of credits! To set up connections and run syncs, <lnk>add credits</lnk>.",
  "credits.creditsProblem.negative_max_threshold": "Youâ€™re out of credits! To set up connections and run syncs, <lnk>add credits</lnk>.",
  "credits.emailVerificationRequired": "You need to verify your email address before you can buy credits.",
  "credits.emailVerification.resendConfirmation": "We send you a new verification link.",
  "credits.emailVerification.resend": "Send verification link again",

  "firebase.auth.error.invalidPassword": "Incorrect password",
  "firebase.auth.error.networkRequestFailed": "There appears to be a network issue. Please try again later.",
  "firebase.auth.error.tooManyRequests": "Too many retries. Please wait 10 minutes before trying again.",
  "firebase.auth.error.default": "Confirmation email cannot be sent. Please try again later.",

  "signup.password.minLength": "Password should be at least 12 characters",
  "email.duplicate": "Email already exists",
  "email.notfound": "Email not found",
  "email.disabled": "Your account is disabled",
  "email.inviteMismatch": "This email does not match the email address sent to this invite.",
  "password.validation": "Your password is too weak",
  "password.invalid": "Invalid password",

  "trial.alertMessage": "You are using a trial of Airbyte. Your trial ends in <b>{value, plural, one {# day} other {# days}}</b>.",

  "verifyEmail.notification": "You successfully verified your email. Thank you.",

  "webapp.cannotReachServer": "Cannot reach server."
}
