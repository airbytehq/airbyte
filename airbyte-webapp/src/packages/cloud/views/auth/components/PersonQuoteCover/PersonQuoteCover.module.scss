@use "../../../../../../scss/colors";
@use "../../../../../../scss/fonts";
@use "../../../../../../scss/variables";

.container {
  position: relative;
  height: 100vh;
}

.image {
  position: absolute;
  top: 0;
  height: 100%;
  width: 100%;
  background-image: url(../../../../../../../public/images/testimonials/cartdotcom-person-photo.png);
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  filter: grayscale(1);
}

.overlay {
  position: absolute;
  height: 100%;
  width: 100%;
  background-image: linear-gradient(
    45deg,
    colors.$orange-400 10%,
    rgba(colors.$orange-400, 0.8) 30%,
    rgba(colors.$blue-400, 0.8) 80%,
    colors.$blue-400 100%
  );
  background-repeat: no-repeat;
  background-size: cover;
}

.contentContainer {
  position: absolute;
  top: calc(50% - 200px);
  left: 50%;
  transform: translate(-50%, 0);
  width: calc(100% - 50px);
  max-width: 600px;
  padding: 0 variables.$spacing-2xl 0 variables.$spacing-2xl;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  gap: 10px;

  .quote {
    margin: 0;
    color: colors.$white;
    font-family: fonts.$regularFont;
    font-weight: 400;
    font-size: 17px;
    line-height: 28px;
    text-align: justify;

    &:before {
      content: open-quote;
      font-size: 50px;
      position: absolute;
      top: 0;
    }
    &:after {
      content: close-quote;
      font-size: 50px;
      position: absolute;
      right: variables.$spacing-2xl;
    }
  }

  .companyLogo {
    color: colors.$white;
    height: 110px;
    margin-top: 40px;
    margin-bottom: 40px;
  }

  .quoteAuthorFullName {
    color: colors.$white;
    font-family: fonts.$regularFont;
    font-weight: 500;
    font-size: 14px;
    margin-bottom: 0;
  }

  .quoteAuthorJobTitle {
    color: colors.$white;
    font-family: fonts.$regularFont;
    font-weight: 700;
    font-size: 18px;
    margin: 10px 0;
  }
}
