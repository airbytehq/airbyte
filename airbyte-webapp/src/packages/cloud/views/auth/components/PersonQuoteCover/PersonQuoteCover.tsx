import React from "react";
import { FormattedMessage } from "react-intl";

import styles from "./PersonQuoteCover.module.scss";

export const PersonQuoteCover: React.FC = () => {
  return (
    <div className={styles.container}>
      <div className={styles.image} data-testid="background-image" />
      <div className={styles.overlay} data-testid="gradient-overlay" />
      <div className={styles.contentContainer}>
        <blockquote className={styles.quote} data-testid="quote">
          <p>
            <FormattedMessage id="login.quoteText" />
          </p>
        </blockquote>
        <img
          src="/images/testimonials/cartdotcom-logo.svg"
          className={styles.companyLogo}
          alt=""
          data-testid="quote-company-logo"
        />
        <div>
          <h5 className={styles.quoteAuthorFullName} data-testid="quote-author-full-name">
            <FormattedMessage id="login.quoteAuthor" />
          </h5>
          <h3 className={styles.quoteAuthorJobTitle} data-testid="quote-author-job-title">
            <FormattedMessage id="login.quoteAuthorJobTitle" />
          </h3>
        </div>
      </div>
    </div>
  );
};
