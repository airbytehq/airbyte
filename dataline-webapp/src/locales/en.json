{
  "sidebar.sources": "Sources",
  "sidebar.destination": "Destination",
  "sidebar.help": "Help",
  "sidebar.docs": "Docs",

  "form.continue": "Continue",
  "form.emailOptional": "Your email (optional)",
  "form.email.placeholder": "you@company.com",
  "form.email.error": "This email address doesn’t seem correct.",
  "form.empty.error": "Empty field",
  "form.selectConnector": "Select a connector",
  "form.searchName": "search by name...",
  "form.noResult": "No result",
  "form.sourceName.placeholder": "Your source name",
  "form.destinationName.placeholder": "Your destination name",
  "form.name": "Name",
  "form.sourceName.message": "Pick a name to help you identify this source in Dataline",
  "form.destinationName.message": "Pick a name to help you identify this destination in Dataline",
  "form.sourceType": "Source type",
  "form.destinationType": "Destination type",
  "form.frequency": "Sync frequency",
  "form.frequency.placeholder": "Select a frequency",
  "form.frequency.message": "Set how often Dataline attempts to replicate data.",
  "form.cancel": "Cancel",
  "form.saveChanges": "Save changes",

  "preferences.title": "Specify your preferences",
  "preferences.anonymizeUsage": "Anonymize usage data collection",
  "preferences.collectData": "We collect data only for product improvements, see the <docs>docs</docs>.",
  "preferences.news": "News and feature updates",
  "preferences.security": "Security updates",
  "preferences.anonymizeData": "Anonymize my usage data.",
  "preferences.featureUpdates": "Receive feature updates.",
  "preferences.unsubcribeAnyTime": " You can unsubcribe any time.",
  "preferences.securityUpdates": "Receive emails about security updates.",

  "onboarding.title": "Let’s set up your first connection.",
  "onboarding.subtitle": "Connections are automated data pipelines that sync data from a source to a destination.",
  "onboarding.createSource": "Create a source",
  "onboarding.sourceSetUp": "Set up the source",
  "onboarding.sourceSetUp.buttonText": "Set up source",
  "onboarding.createDestination": "Create a destination",
  "onboarding.destinationSetUp": "Set up the destination",
  "onboarding.destinationSetUp.buttonText": "Set up destination",
  "onboarding.setUpConnection": "Set up connection",
  "onboarding.setConnection": "Set up the connection",
  "onboarding.instructionsLink": "See instructions how to connect {name}",

  "sources.newSource": "+ New source",
  "sources.newSourceTitle": "New Source",
  "sources.selectSource": "Select a source",

  "destination.destinationSettings": "Destination Settings"
}
