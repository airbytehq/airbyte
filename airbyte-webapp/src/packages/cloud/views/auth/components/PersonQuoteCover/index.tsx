import { PersonQuoteCover } from "./PersonQuoteCover";

export default PersonQuoteCover;
