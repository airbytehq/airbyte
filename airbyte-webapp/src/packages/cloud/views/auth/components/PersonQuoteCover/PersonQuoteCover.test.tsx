import { render } from "@testing-library/react";
import { IntlProvider } from "react-intl";

import en from "../../../../../../packages/cloud/locales/en.json";
import { PersonQuoteCover, PersonQuoteCoverProps } from "./PersonQuoteCover";

const renderPersonQuoteCover = (props?: PersonQuoteCoverProps) =>
  render(
    <IntlProvider locale="en" messages={en}>
      <PersonQuoteCover {...props} />
    </IntlProvider>
  );

describe("<PersonQuoteCover />", () => {
  test("should render with default props", () => {
    const { asFragment } = renderPersonQuoteCover();

    expect(asFragment()).toMatchSnapshot();
  });

  test("should render with default props and without background image", () => {
    const { asFragment } = renderPersonQuoteCover({
      backgroundImageStyle: {
        backgroundImage: "none",
      },
    });

    expect(asFragment()).toMatchSnapshot();
  });

  test("should render with overridden props", () => {
    const { asFragment } = renderPersonQuoteCover({
      overlayGradientStyle: {
        //  https://github.com/jsdom/cssstyle/issues/148
        //  snapshot testing doesn't parse linear-gradient so for testing purpose we just check applying inline style
        background: "blue",
      },
      backgroundImageStyle: {
        backgroundImage: "url(/image.png)",
      },
      quoteText: "This is test text",
      quoteTextStyle: { fontSize: "30px" },
      logoImageSrc: "/new-logo.svg",
      quoteAuthorFullNameStyle: { color: "black", fontSize: "40px" },
      quoteAuthorJobTitleStyle: { color: "red", fontSize: "50px" },
    });

    expect(asFragment()).toMatchSnapshot();
  });
});
