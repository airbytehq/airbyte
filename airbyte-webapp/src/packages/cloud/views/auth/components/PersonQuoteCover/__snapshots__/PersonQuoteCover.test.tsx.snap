// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PersonQuoteCover /> should render with default props 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="image"
      data-testid="background-image"
    />
    <div
      class="overlay"
      data-testid="gradient-overlay"
    />
    <div
      class="contentContainer"
    >
      <blockquote
        class="quote"
        data-testid="quote"
      >
        <p>
          Airbyte has cut &lt;b&gt;months of man hours off&lt;/b&gt; of our ELT pipeline development and delivered usable data to us in hours instead of weeks. We are excited for the future of Airbyte, enthusiastic about their approach, and optimistic about our future together.
        </p>
      </blockquote>
      <img
        alt=""
        class="companyLogo"
        data-testid="quote-company-logo"
        src="/images/testimonials/cartdotcom-logo.svg"
      />
      <div>
        <h5
          class="quoteAuthorFullName"
          data-testid="quote-author-full-name"
        >
          Micah Mangione
        </h5>
        <h3
          class="quoteAuthorJobTitle"
          data-testid="quote-author-job-title"
        >
          Director of Technology
        </h3>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`<PersonQuoteCover /> should render with default props and without background image 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="image"
      data-testid="background-image"
      style="background-image: none;"
    />
    <div
      class="overlay"
      data-testid="gradient-overlay"
    />
    <div
      class="contentContainer"
    >
      <blockquote
        class="quote"
        data-testid="quote"
      >
        <p>
          Airbyte has cut &lt;b&gt;months of man hours off&lt;/b&gt; of our ELT pipeline development and delivered usable data to us in hours instead of weeks. We are excited for the future of Airbyte, enthusiastic about their approach, and optimistic about our future together.
        </p>
      </blockquote>
      <img
        alt=""
        class="companyLogo"
        data-testid="quote-company-logo"
        src="/images/testimonials/cartdotcom-logo.svg"
      />
      <div>
        <h5
          class="quoteAuthorFullName"
          data-testid="quote-author-full-name"
        >
          Micah Mangione
        </h5>
        <h3
          class="quoteAuthorJobTitle"
          data-testid="quote-author-job-title"
        >
          Director of Technology
        </h3>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`<PersonQuoteCover /> should render with overridden props 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="image"
      data-testid="background-image"
      style="background-image: url(/image.png);"
    />
    <div
      class="overlay"
      data-testid="gradient-overlay"
      style="background: blue;"
    />
    <div
      class="contentContainer"
    >
      <blockquote
        class="quote"
        data-testid="quote"
        style="font-size: 30px;"
      >
        <p>
          This is test text
        </p>
      </blockquote>
      <img
        alt=""
        class="companyLogo"
        data-testid="quote-company-logo"
        src="/new-logo.svg"
      />
      <div>
        <h5
          class="quoteAuthorFullName"
          data-testid="quote-author-full-name"
          style="color: black; font-size: 40px;"
        >
          Micah Mangione
        </h5>
        <h3
          class="quoteAuthorJobTitle"
          data-testid="quote-author-job-title"
          style="color: red; font-size: 50px;"
        >
          Director of Technology
        </h3>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
